src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -473,6 +473,24 @@ private CandidateStatus getCandidateStatus(String candidate) {
       }
     }
 
+    // Edge case: if we've identified multiple winners in this round but we're only supposed to
+    // elect one winner per round, pick the top vote-getter and defer the others to subsequent
+    // rounds.
+    if (config.isOnlyOneWinnerPerRoundEnabled() && selectedWinners.size() > 1) {
+      BigDecimal maxTally = BigDecimal.ZERO;
+      String candidateWithMaxTally = null;
+      for (String candidate : selectedWinners) {
+        BigDecimal tally = currentRoundCandidateToTally.get(candidate);
+        // TODO: need a non-arbitrary way of settling a tie here
```

Comment:
Definitely need to address this before submitting this PR. Maybe email George and the others directly to get a quicker response?